tests/js/e2e_tests/manageSubProject.js

```diff
@@ -78,8 +78,8 @@ describe("manageSubProject", function() {
 
     browser.get('manageSubProject.php?projectid=8#/groups');
     var rows = element(by.tagName('tbody')).all(by.tagName('tr'));
-    expect(rows.get(0).element(by.name("group_name")).getAttribute("value")).toBe("gorup2");
-    expect(rows.get(1).element(by.name("group_name")).getAttribute("value")).toBe("group1");
+    expect(rows.get(0).element(by.name("group_name")).getAttribute("value")).toBe("group1");
+    expect(rows.get(1).element(by.name("group_name")).getAttribute("value")).toBe("gorup2");
```

Comment:
Yeah, it gets renamed as part of the test.
https://github.com/Kitware/CDash/blob/master/tests/js/e2e_tests/manageSubProject.js#L76